servlet/src/test/java/io/undertow/servlet/test/defaultservlet/DefaultServletCachedResourceTestCase.java

```diff
@@ -0,0 +1,302 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2014 Red Hat, Inc., and individual contributors
```

Comment:
2014... Someone really needs to update his IDE ;-)

Reply:
It was a copy/paste